src/pages/components/notification.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Notification
+description: >
+  A Notification is a passive status update that keeps users informed of system progress.
+package: '@zendeskgarden/react-notifications'
+components:
+  - notifications/src/elements/content/Close.tsx
+  - notifications/src/elements/Notification.tsx
+  - notifications/src/elements/content/Paragraph.tsx
+  - notifications/src/elements/content/Title.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>For a passive status update about user or system activity</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        <Markdown>
+          For urgent, inline feedback that may require action or decision, use an
+          [Alert](/components/alert) instead.
+        </Markdown>
+      </li>
+    </ul>
+  </Misuse>
+</Usage>
+
+## How to use it
+
+### Default
+
+The typical usage of a Notification component.
+
+import Default from '../../examples/notifications/notification/NotificationDefault.tsx';
+import DefaultCode from '!!raw-loader!../../examples/notifications/notification/NotificationDefault.tsx';
+
+<CodeExample code={DefaultCode}>
+  <Default />
+</CodeExample>
+
+### Type
+
+Notification types help communicate the tone of the message. They can be
```

Comment:
Like above: 
`Types indicate what kind of message is in the alert and help the user understand how to respond.`